docs/content/latest/yugabyte-cloud/cloud-monitor/overview.md

```diff
@@ -21,35 +21,56 @@ Monitor performance metrics for your cluster to ensure the cluster configuration
 
 Use these metrics to monitor the performance of your cluster and to determine whether the configuration needs to change. For information on changing or scaling your cluster, refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
 
+You can enable alerts for some performance metrics. Refer to [Cloud alerts](../cloud-alerts/).
+
 ![Cloud Cluster Performance Metrics](/images/yb-cloud/cloud-clusters-metrics.png)
 
-You can show metrics for the past hour, 6 hours, 12 hours, 24 hours, or 7 days. The **Overview** tab shows metrics averaged over all the nodes in the cluster. On the **Performance** tab you can additionally view the metrics for specific nodes.
+You can show metrics for the past hour, 6 hours, 12 hours, 24 hours, or 7 days.
+
+## Overview metrics
+
+The **Overview** tab shows metrics averaged over all the nodes in the cluster.
+
+You can enable alerts for CPU usage and disk usage. Refer to [Cloud alerts](../cloud-alerts/).
 
 The following table describes the metrics available on the **Overview**.
 
-| Graph | **Description** | **Use** |
+| Graph | Description | Use |
 | :---| --- | --- |
```

Comment:
Always good to be explicit with the alignment. Markdown tables are dumb enough on their own without us helping them be worse!

```suggestion
| :--- | :--- | :--- |
```